ci/aws-rds-storage.yml

```
@@ -3,9 +3,11 @@ platform: linux
 image_resource:
   type: registry-image
   source:
-    repository: kennethreitz/pipenv
-    registry_mirror:
-      host: docker-registry-mirror.app.cloud.gov:443
+    aws_access_key_id: ((ecr_aws_key))
+    aws_secret_access_key: ((ecr_aws_secret))
+    repository: harden-concourse-task
```

```suggestion
    repository: general-task
```

I think the general-task image is what we want to use for most things now